doc/cartridge_kubernetes_guide/index.rst

```diff
@@ -0,0 +1,1016 @@
+
+Wait for the cluster state to be *Ready*:
+
+.. code-block:: console
+
+   $ kubectl get nodes
+   ---
+   NAME       STATUS   ROLES    AGE   VERSION
+   minikube   Ready    master   21m   v1.16.4
+
+.. _cartridge_kubernetes_creating_cluster_kind:
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Using *kind*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Create a Kubernetes cluster of version 1.16.4 by using the *kind* utility as an
+alternative to *minikube*:
+
+.. code-block:: console
+
+   $ kind create cluster  --image kindest/node:v1.16.4
+   ---
+   Creating cluster "kind" ...
+    ✓ Ensuring node image (kindest/node:v1.16.4) 🖼
+    ✓ Preparing nodes 📦
+    ✓ Writing configuration 📜
+    ✓ Starting control-plane 🕹️
+    ✓ Installing CNI 🔌
+    ✓ Installing StorageClass 💾
+   Set kubectl context to "kind-kind"
+   You can now use your cluster with:
+
+   kubectl cluster-info --context kind-kind
+
+   Not sure what to do next? 😅  Check out https://kind.sigs.k8s.io/docs/user/quick-start/
+
+Let's check the cluster status:
+
+.. code-block:: console
+
+   $ kubectl get nodes
+   ---
+   NAME                 STATUS   ROLES    AGE   VERSION
+   kind-control-plane   Ready    master   48s   v1.16.4
+
+.. _cartridge_kubernetes_launch_the_application:
+
+--------------------------------------------------------------------------------
+Launch application
+--------------------------------------------------------------------------------
+
+To install Tarantool Kubernetes operator and deploy the cluster, we will use
+the ``helm`` utility. Charts are published in our repository. Let’s add it:
+
+.. code-block:: console
+
+   $ helm repo add tarantool https://tarantool.github.io/tarantool-operator
+
+Two charts are available in the repository:
+
+.. code-block:: console
+
+   $ helm search repo tarantool
+   ---
+   NAME                            CHART VERSION   APP VERSION DESCRIPTION
+   tarantool/tarantool-operator    0.0.8           1.16.0      kubernetes tarantool operator
+   tarantool/cartridge             0.0.8           1.0         A Helm chart for tarantool
+
+The ``tarantool/tarantool-operator`` chart installs and configures the
+operator that manages Tarantool Cartridge clusters.
+
+The ``tarantool/cartridge`` chart is a template for creating Tarantool
+Cartridge clusters. With the default settings, this chart deploys an
+example application consisting of 3 instances. The chart works only in
+conjunction with the Tarantool Kubernetes operator.
+
+.. NOTE::
+
+   Use the same version of the charts. If you set the ``tarantool-operator``
+   chart to version 0.0.8, set the ``cartridge`` chart to the same version 0.0.8.
+
+Install *tarantool-operator* in the *tarantool* namespace:
+
+.. code-block:: console
+
+   $ helm install tarantool-operator tarantool/tarantool-operator --namespace tarantool --create-namespace --version 0.0.8
```

Comment:
Gives me a bunch of these errors:
```
W0107 10:23:23.531224   28894 warnings.go:67] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
```
Is this error on our side?

Reply:
No, this is not our fault, but it is worth paying attention.